VK_App/VK_App/View/UserGroupViewCell.swift

```diff
@@ -10,16 +10,10 @@ final class UserGroupViewCell: UITableViewCell {
     @IBOutlet private var userGroupNameLabel: UILabel!
     @IBOutlet private var userGroupImageView: UIImageView!
 
-    // MARK: - Private properties
-
-    private var networkService = NetworkService()
-
     // MARK: - Public properties
 
-    func configure(_ group: Group) {
+    func configure(_ group: Group, photo: UIImage?) {
```

Comment:
```suggestion
    func configure(_ group: Group, photo: ) {
```
аналогично не передаем UIImage